public/app/features/dashboard/components/GenAI/GenAIButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/css';
```

Comment:
Naming is hard... wasnt sure to keep with `DashGPT` or just try to make it more general to generative AI - i dont feel strongly either way so please feel free to modify any and all naming 😅